ante/fee.go

```diff
@@ -89,23 +92,40 @@ func IsOracleOrGravityTx(msgs []sdk.Msg) bool {
 
 // getTxPriority returns naive tx priority based on the lowest fee amount (regardless of the
 // denom) and oracle tx check.
-func getTxPriority(fee sdk.Coins, isOracleOrGravity bool) int64 {
+func getTxPriority(fees sdk.Coins, gasAmount int64, isOracleOrGravity bool, msgs []sdk.Msg) int64 { //nolint: unparam
```

Comment:
Commented code here is a bit messy, maybe we could just put this code in the issue?

At the very least let's take out the unused parameter
```suggestion
func getTxPriority(gasAmount int64, isOracleOrGravity bool, msgs []sdk.Msg) int64 {
```

Reply:
i was thinking to keep the `fees` parameter for the followup task (to have more general tx prioritization once IBC issue will be resolved).

Reply:
I'm not super against it as long as it's cleaned up before v3 gets released

Reply:
We won't update it before v3. OK, I will remove the argument, but leave the commented code below.